meganav.yaml

```
@@ -1,3 +1,699 @@
+              url: /engage/robot-operating-system-choice
+            - title: "Embedded Linux: make or buy?"
+              url: /engage/embedded-linux-make-or-buy
+            - title: Try Ubuntu Core on Raspberry Pi
+              url: /download/raspberry-pi-core
+            - title: Industry 4.0
+              url: /engage/bridge-the-IT-OT-gap-Industry4
+
+use_case:
+  primary_links:
+    - title: By solution
+      links:
+        - title: AI/ML
+          url: /ai
+        - title: Compliance
+          url: /security/certifications
+        - title: Containers
+          url: /containers
+        - title: IoT
+          url: /internet-of-things
+        - title: Edge computing
+          url: /edge-computing
+        - title: Managed applications
+          url: /managed/apps
+        - title: Managed infrastructure
+          url: /pricing/infra
+        - title: Observability
+          url: /observability
+        - title: Private cloud and data centre
+          url: /openstack/consulting
+        - title: Public cloud
+          url: /cloud/public-cloud
+        - title: Robotics
+          url: /robotics
+        - title: Security
+          url: /security
+
+    - title: By Industry
+      links:
+        - title: Telco
+          url: /telco
+        - title: Automotive
+          url: /automotive
+        - title: Financial services
+          url: /financial-services
+        - title: Industrial
+          url: /industrial
+  secondary_links:
+    - title: IoT/Devices Use Cases/Verticals
+      links:
+        - title: Robotics
+          url: /robotics
+        - title: EdgeX
+          url: /internet-of-things/edgex
+        - title: Smart Home
+          url: /internet-of-things/smart-home
+        - title: Industrial
+          url: /industrial
+        - title: Automotive
+          url: /automotive
+        - title: Smart displays
+          url: /internet-of-things/smart-displays
+
+support:
+  side_nav_sections:
+    - title: Enterprise
+      primary_links:
+        - title: Products
+          links:
+            - title: Ubuntu Pro
+              description: Performant open source
+              url: /support
+            - title: Ubuntu Pro Azure
+              description: Support in the cloud
+              url: /azure/pro
+            - title: Ubuntu Pro AWS
+              description: Support in the cloud
+              url: /aws/pro
+            - title: Ubuntu Pro GCP
+              description: Enterprise support for Ubuntu
+              url: /blog/introducing-ubuntu-pro-for-google-cloud
+            - title: Managed apps
+              description: Support for open infrastructure
+              url: /managed/apps
+            - title: Managed Infrastructure
+              url: /pricing/infra
+      secondary_links:
+        - links:
+            - title: Expanded Security Maintenance
+              url: /security/esm
+            - title: Pricing
+              url: /pricing/infra
```

```suggestion
              url: /pricing/pro
```
This redirects to /pro